packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -1467,8 +1467,9 @@ const registerNetworkIpc = (mainWindow) => {
         );
       };
 
+      const dirPath = path.dirname(pathname);
       const fileName = determineFileName();
-      const filePath = await chooseFileToSave(mainWindow, fileName);
+      const filePath = await chooseFileToSave(mainWindow, `${dirPath}/${fileName}`);
```

Comment:
This may fail on windows. I believe `path.join` should make this work for all OSes